portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsTranslations.testcase

```diff
@@ -142,14 +142,62 @@ definition {
 	}
 
 	@description = "Verify that Forms submitted in different languages show the appropriate country flags in View Entries"
-	@ignore = "Test Stub"
 	@priority = "5"
 	test EntriesDisplayTheCorrectLanguageFlag {
 		property forms.functional = "true";
 		property portal.acceptance = "false";
 
-		// TODO LPS-133523 EntriesDisplayTheCorrectLanguageFlag pending implementation
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			portlet = "Forms");
+
+		FormsAdminNavigator.gotoAddForm();
+
+		Form.editName(formName = "Form 1");
+
+		Form.gotoAddField(
+			fieldPositionNumber = "0",
+			fieldType = "Text");
+
+		Form.addTranslation(localizedLanguage = "Spanish (Spain)");
+
+		Click(
+			key_name = "Text",
+			locator1 = "FormFields#FIELD_TARGET_LABEL");
+
+		Form.editFieldLabelMetal(fieldValue = "Text ES");
+
+		Form.save();
+
+		Form.publishForm();
+
+		FormsAdminNavigator.gotoPublishedForm();
+
+		FormViewBuilder.changeAppLanguageTo(changeAppLanguageTo = "en-US");
+
```

Comment:
Hi @chaowang-123, 

According to the [test scenario](https://issues.liferay.com/browse/LPS-133523), this step doesn't exit. Please, could remove this line?
`FormViewBuilder.changeAppLanguageTo(changeAppLanguageTo = "en-US");`

>

Reply:
Sorry, I've found that this line is really necessary. =)